to-statsd.js

```diff
@@ -8,7 +8,7 @@ module.exports = function messageToStatsd(message)
 	{
 		if (key.match(/(time|value|name|timing|set|count|rate)/)) return '';
 		const val = message[key];
-		if (!val || val instanceof Object) return ''; // only want strings, nums, or bools
+		if (!['string', 'number', 'boolean'].includes(typeof val)) return ''; // only want strings, nums, or bools
```

Comment:
:clap: look at that, it actually does the exact thing the comment says it does

Reply:
Wild, right? 😄